README.md

```diff
@@ -26,15 +26,33 @@
 
 ## Description
 
-Service to receive and convert Alerts from the Prometheus Alertmanager and convert into own LogFormat.
+Service to receive and convert Alerts from the Prometheus Alertmanager and convert into own LogFormat. To use this Service install Prometheus, Alertmanager and Windows-Exporter
+and replace the config with the one in this Repository. 
 
 Port : localhost:3900
 
+[GET]
+
+At /get-sample you receive an Array of example LogMessages with are converted from sample-alert.json.
+
+[POST]
+
+At /post-alerts you can post a Alert in the format of the Alertmanager, it will be converted into a LogMessage and send to the Queue.
+
+
 ## Installation
+### Installing the local Repository:
 
 ```bash
 $ npm install
 ```
+### Install Prometheus, Alertmanager and Windows-Exporter:
+
+Download Prometheus and Alertmanager from https://prometheus.io/download/
+
+and Windows-Exporter from: https://github.com/prometheus-community/windows_exporter/releases
+
+then replace the prometheus.yml and alert-manager.yml with the ones of this Repository and add the rule.yml to your prometheus folder.
```

Comment:
eindeutige pfade wäre hier sehr cool. außerdem würde ich die pfade hervorheben, z.b. als `inline code`.

Reply:
Was meinst du genau mit eindeutige Pfade?

Reply:
z.B. `./path/to/file/prometheus.yml`
einfach den ordner mit angeben, damit man nicht suchen muss, wo genau die datein liegen.

Reply:
naja den hab ich ja nicht, das muss man ja selber runterladen.

Reply:
eh? und was für datein sind dann mit 'the ones of this Repository' gemeint? o_O

Reply:
Achso ja die hab ich verlinkt, passt das auch so? Hab gerade gepusht.